### ISSUE TITLE
Unable to read deltatables with binary columns: Binary is not supported by JSON

### ISSUE THREAD

**Issue by johnnyg** [delta-io/delta-rs#2302]
# Environment

**Delta-rs version**: 0.16.1

**Binding**: Python

**Environment**:
- **Cloud provider**: AWS
- **OS**: Linux

***
# Bug

**What happened**:

Tried to read a deltatable with binary columns but get the following error:
```python
  File "/home/user/mamba/envs/dev/lib/python3.11/site-packages/deltalake/table.py", line 408, in __init__
    self._table = RawDeltaTable(
                  ^^^^^^^^^^^^^^
Exception: Json error: Binary is not supported by JSON
```

**What you expected to happen**:

To be able to read the table (and its schema)

**How to reproduce it**:

Have a deltatable with a binary column i.e. the schema type for a field is "binary"

**Comment by leoeareis** [comment]
I'm facing the same problem :disappointed:

**Comment by ion-elgreco** [comment]
@johnnyg or @leoeareis can you share some reproducible code that creates a delta table with binary column? Then I  can have a look at it

**Comment by flxliu** [comment]
@ion-elgreco : I filed a different bug https://github.com/delta-io/delta-rs/issues/2373, it has the same error message. it can repro this bug.

**Comment by johnnyg** [comment]
This appears to be fixed in 0.16.4 so I'm going to close this ticket.
